src/example_package/flask_example.py

```diff
@@ -0,0 +1,38 @@
+from flask import Flask
+
+app = Flask(__name__)
+
+# view function: area "homepage"
+@app.route("/") # URL
+def homepage():
+    hello = ("<h1 style='color:red'>" +
+             "Hello World!" +
+             "</h1>" +
+             
+             "<h2 style='color:green'>" +
+             "Flask funziona correttamente." +
+             "</h2>" +
+             
+             "<h3 style='color:blue'>" +
+             "Visita il sito: \"./contatti\"." +
+             "</h3>")
+
+    return hello
+
+# view function: area "contatti"
+@app.route("/contatti") # URL
```

Comment:
'/contatti' is this an html response right? why? is it really needed?
please consider to use a JSONResponse
or please jsonify your response